editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/features/custom/GoIntoContainerFeature.java

```diff
@@ -75,16 +75,15 @@ private boolean isGoUp(AbstractCamelModelElement cme, CamelDesignEditor editor)
 		// go up is allowed if:
 		// - selected container is a route
 		// - selected container is the selected element
-		return  editor.getSelectedContainer().equals(cme) && 
-				cme instanceof CamelRouteElement;
+		return cme instanceof CamelRouteElement && cme.equals(editor.getSelectedContainer());
 	}
 	
 	private boolean isGoInto(AbstractCamelModelElement cme, CamelDesignEditor editor) {
 		// go into is allowed if:
 		// - selected element is a route
 		// - selected container is the camel context
 		// - context contains more than one route
-		return 	cme.getNodeTypeId().equalsIgnoreCase("route") && 
+		return 	"route".equalsIgnoreCase(cme.getNodeTypeId()) && 
```

Comment:
Do we want to externalize this string or mark it as not externalizable? (i.e.  //$NON-NLS-1$ )